lib/evmone/instructions.hpp

```diff
@@ -783,6 +783,49 @@ inline void gas(StackTop stack, ExecutionState& state) noexcept
     stack.push(state.gas_left);
 }
 
+namespace
+{
+
+template <size_t Len>
+inline uint64_t load_partial_push_data(code_iterator pos) noexcept
+{
+    static_assert(Len > 4 && Len < 8);
+
+    auto d = intx::be::unsafe::load<uint64_t>(pos);
```

Comment:
The `intx::` is not needed. Also drop temporary variables.

Reply:
`intx::` is needed.